code/__DEFINES/atmospherics/atmos_mob_interaction.dm

```diff
@@ -59,10 +59,14 @@
 /// This also affects how fast the body normalises it's temperature when hot.
 /// 340k is about 66c, and rather high for a human.
 #define BODYTEMP_HEAT_DAMAGE_LIMIT (BODYTEMP_NORMAL + 30)
+/// A temperature limit which is above the maximum lavaland temperature
+#define BODYTEMP_HEAT_LAVALAND_SAFE (BODYTEMP_HEAT_DAMAGE_LIMIT + 20)
 /// The body temperature limit the human body can take before it starts taking damage from cold.
```

Comment:
if you're going to make this a define, could you ensure it's tied to atmosphere generation figures somehow? or at least throw an error if it goes below that threshold